opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json

```diff
@@ -167,18 +176,168 @@
+        "openmrs_entity_id": "114403AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+        "openmrs_entity_parent": "",
+        "label": "Je, katika siku 3 zilizopita, umempeleka mtoto kwenye kituo cha afya kwa mojawapo ya hali hizi zilizotambuliwa?",
+        "options": [
+          {
+            "key": "yes",
+            "openmrs_entity": "",
+            "openmrs_entity_id": "1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+            "openmrs_entity_parent": "",
+            "text": "Ndio"
+          },
+          {
+            "key": "no",
+            "openmrs_entity": "",
+            "openmrs_entity_id": "1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+            "openmrs_entity_parent": "",
+            "text": "Hapana"
+          }
+        ],
+        "relevance": {
+          "step1:danger_signs_present_child": {
+            "ex-checkbox": [
+              {
+                "or": [
+                  "chk_fever",
+                  "chk_excessive_crying",
+                  "chk_fast_breathing",
+                  "chk_difficult_breathing",
+                  "chk_unable_to_suck",
+                  "chk_umbilical_discharge",
+                  "chk_convulsions",
+                  "chk_skin_rashes",
+                  "chk_pale",
+                  "chk_severe_abdominal_pain",
+                  "chk_lethargy",
+                  "chk_urination"
+                ]
+              }
+            ]
+          }
+        },
+        "v_required": {
+          "value": "true",
+          "err": "Tafadhali chagua chaguo"
+        }
+      },
+      {
+        "key": "discuss_neonatal_advice_health_facility",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "",
+        "type": "toaster_notes",
+        "text": "\nJadili na mama kuhusu ushauri alioupata kwa masharti haya alipompeleka Mtoto katika kituo cha Afya ikiwa ni pamoja na Ushauri Nasaha, matibabu wakati wa kufuatilia.",
+        "toaster_type": "info",
+        "relevance": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "pnc_danger_signs_baby_relevance.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "issue_referral_neonatal",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "",
+        "type": "toaster_notes",
+        "text": "\nUnapaswa kumpeleka Mtoto kwenye kituo cha afya haraka iwezekanavyo na ndani ya saa 24 kwa uchunguzi zaidi.",
+        "text_color": "#CF0800",
+        "toaster_type": "problem",
+        "relevance": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "pnc_danger_signs_baby_relevance.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "neonatal_referral_health_facility",
+        "type": "native_radio",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "114403AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```

Comment:
We are phasing out the use of openmrs uuids here, please add the same value as the key